string[] name
std_msgs/Header header
float64[] xPosition
float64[] yPosition
float64[] zPosition
float64[] xVelocity
float64[] yVelocity
float64[] zVelocity